be/src/storage/rowset/segment_v2/segment.h

```diff
@@ -187,6 +187,7 @@ class Segment : public std::enable_shared_from_this<Segment> {
     bool _needs_chunk_adapter = false;
     // When the storage types is different with TabletSchema
     bool _needs_block_adapter = false;
+    uint32_t _mem_usage = 0;
```

Comment:
Please place `_mem_usage` above the boolean variables for less memory usage

Reply:
Fixed